### PR TITLE
Display categories in menu

### PR DESCRIPTION
**Pull request by SophieSjoberg**
As a customer
In order to choose more easily
I would like to see the dishes sorted in categories

https://app.zenhub.com/workspace/o/craftacademy/slow_food_sinatra_mvc_lions_and_crowns/issues/9

### CODE REVIEW

File: app/views/welcome.erb [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
-  <h1>
-    <%= "#{product.name} - #{product.price} kr" %>
-  </h1>
-  <p>
-    <%= product.description %>
-  </p>
-
+<% @categories.each do |category| %>
+  <section id="<%= category.titleize.gsub(' ', '').underscore %>">
+    <% @products.each do |product| %>
```
Comment (tochman) [inline app/views/welcome.erb]: You lo-fi states that the category name should be displayed, right? Please add it here.

File: features/dish_in_categories_on_landing_page.feature [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      | Apple Pie | 40 | Great Apple Pie | Dessert |
+      | Nachos | 60 | Great Nachos | Starter |
+
+  Scenario: Customers see dishes sorted in to categories on the page
+    When I visit the site
+    Then I should see "Pizza" in the "Main Course" section
+    And I should see "Apple Pie" in the "Dessert" section
+    And I should see "Nachos" in the "Starter" section
+
+
```
Comment (tochman) [inline features/dish_in_categories_on_landing_page.feature]: Please remove empty lines

**Review by tochman — CHANGES_REQUESTED** [review]

File: app/views/welcome.erb (hunk 1 above)
Comment (SophieSjoberg) [reply app/views/welcome.erb]: Will do boss!

File: features/dish_in_categories_on_landing_page.feature (hunk 2 above)
Comment (SophieSjoberg) [reply features/dish_in_categories_on_landing_page.feature]: Will do boss!

File: app/views/welcome.erb (hunk 1 above)
Comment (SophieSjoberg) [reply app/views/welcome.erb]: Done! Better?

File: features/dish_in_categories_on_landing_page.feature (hunk 2 above)
Comment (SophieSjoberg) [reply features/dish_in_categories_on_landing_page.feature]: Done

File: app/views/welcome.erb (hunk 1 above)
Comment (tochman) [reply app/views/welcome.erb]: Where did you add this? Can't see it.

File: features/dish_in_categories_on_landing_page.feature (hunk 2 above)
Comment (tochman) [reply features/dish_in_categories_on_landing_page.feature]: Not removed. Or not pushed up ;-)

File: app/views/welcome.erb (hunk 1 above)
Comment (SophieSjoberg) [reply app/views/welcome.erb]: Now?

File: app/views/welcome.erb [hunk 3]
```diff
@@ -1,11 +1,18 @@
...
-    <%= "#{product.name} - #{product.price} kr" %>
-  </h1>
-  <p>
-    <%= product.description %>
-  </p>
-
+<% @categories.each do |category| %>
+  <section id="<%= category.titleize.gsub(' ', '').underscore %>">
+    <h1>
+      <%= "#{category.pluralize}" %>
```
Comment (tochman) [inline app/views/welcome.erb]: No need to use string interpolation here. Just the variable

File: features/list_dishes_on_landing_page.feature [hunk 4]
```diff
@@ -23,3 +23,4 @@ Feature: List dishes on landing page
     And I would like to see "Great Apple Pie"
     And I would like to see "Cheese Cake" with a price of "60" kr
     And I would like to see "Great Cheese Cake"
+    Then show me the page
```
Comment (tochman) [inline features/list_dishes_on_landing_page.feature]: Please remove this. No debugger steps are needed when you are done with the feature

Comment (SophieSjoberg) [reply features/list_dishes_on_landing_page.feature]: Done

File: app/views/welcome.erb (hunk 1 above)
Comment (SophieSjoberg) [reply app/views/welcome.erb]: Done

**Review by tochman — APPROVED** [review]

**Pull request merged**